benchmarks/Microsoft.AspNetCore.Routing.Performance/Matching/JumpTableMultipleEntryBenchmark.cs

```diff
@@ -164,7 +155,7 @@ private static string[] GetStrings(int count)
                 var guid = Guid.NewGuid().ToString();
 
                 // Between 5 and 36 characters
-                var text = guid.Substring(0, Math.Max(5, Math.Min(count, 36)));
+                var text = guid.Substring(0, Math.Max(5, Math.Min(i, 36)));
```

Comment:
This was always generating the same thing length, which wasn't intended, now it generates a range of different string lengths.